engine/release-notes/index.md

```diff
@@ -15,6 +15,31 @@ for Docker Engine.
 
 # Version 20.10
 
+## 20.10.19
+2022-10-14
+
+This release of Docker Engine comes with some bug-fixes, and an updated version
+of Docker Compose.
+
+### Builder
+
+- Fix an issue that could result in a panic during `docker builder prune` or
+  `docker system prune` [moby/moby#44122](https://github.com/moby/moby/pull/44122){:target="_blank" rel="noopener"}.
+
+### Daemon
+
+- Fix an bug where using `docker volume prune` would remove volumes that were
```

Comment:
```suggestion
- Fix a bug where using `docker volume prune` would remove volumes that were
```

Reply:
Ah, thanks! I guess I had `Fix an issue` before, and changed it to bug 😅 

Either way; updated, PTAL 🤗